Examples/Swift/ViewController.swift

```diff
@@ -148,6 +148,12 @@ class ViewController: UIViewController, MGLMapViewDelegate, NavigationViewContro
         navigationViewController.voiceController?.volume = 0.5
         navigationViewController.navigationDelegate = self
         
+        // Uncomment to apply custom styles
+//        styleForRegular().apply()
+//        styleForCompact().apply()
+//        styleForiPad().apply()
+//        styleForCarPlay().apply()
```

Comment:
This approach accomplishes the goal of allowing developers to theme the navigation UI and even vary it by size class. However, I wonder if we can make more direct use of the UIAppearance system.

Since RouteViewController is the entry point for the entire SDK’s UI, would it make sense to encapsulate all these style properties within that class? RouteViewController would expose properties like `maneuverViewHeight` by passing those values along to the maneuver view. The developer would call `appearance(for:)` directly on RouteViewController in order to set these properties. This would allow us to avoid making classes like SeparatorView public and also make it easier to use a single appearance proxy for the entire themeable interface.

Down the line, if we decide to make ManeuverView public and more customizable, for example, we could expose a `maneuverView` property on RouteViewController; since UIView already conforms to UIAppearanceContainer, we’d get themeability “for free”.